pywinauto/hooks.py

```diff
@@ -0,0 +1,317 @@
+from ctypes import wintypes
+from ctypes import windll, CFUNCTYPE, POINTER, c_int, c_uint, c_long, c_longlong, c_ulong, c_void_p, byref, sizeof
+from ctypes import Structure
+import atexit
+
+if sizeof(POINTER(c_int)) * 8 == 64:
+    hinstance = c_longlong
+else:
+    hinstance = c_long
+
+cmp_func = CFUNCTYPE(c_int, c_int, hinstance, POINTER(c_void_p))
+DWORD = c_ulong
+
+windll.kernel32.GetModuleHandleA.restype = wintypes.HMODULE
+windll.kernel32.GetModuleHandleA.argtypes = [wintypes.LPCWSTR]
+
+windll.user32.SetWindowsHookExA.restype = c_int
+windll.user32.SetWindowsHookExA.argtypes = [c_int, cmp_func, hinstance, DWORD]
+
+
+class MSG(Structure):
```

Comment:
I didn't try `wintypes.MSG` but agree with @airelil: we need to re-use maximum of `wintypes` functionality since it's so rich and built-in (i.e. no 3rd party dependency).